config.yml

```diff
@@ -94,3 +95,11 @@ default:
   scenario_sources_list: ["GECO2022", "IPR2021", "ISF2021", "WEO2022"]
   scenario_raw_data_to_include: ["geco_2022", "ipr_2021", "isf_2021", "weo_2022"]
   global_aggregate_scenario_sources_list: ["WEO2022"]
+
+2022Q4_CICD:
```

Comment:
what does CICD stand for?

Reply:
Continuous Integration/Continuous Deployment

Reply:
I'm not married to the name

Reply:
Just curious